tests/cephadm/test_legacy_services.py

```diff
@@ -0,0 +1,51 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Verify if the cephadm shell tries to infer config from
+    a dummy mon created.
+    Verifies bz:2080242
+
+    Args:
+        ceph_cluster: Ceph cluster object
+        **kw :     Key/value pairs of configuration information to be used in the test.
+
+    Returns:
+        int - 0 when the execution is successful else 1 (for failure).
+    """
+
+    config = kw.get("config")
+
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+
+    # Creating a dummy mon on a non monitor node
+    for node in cephadm.cluster.get_nodes():
+        if node.role != "mon" or node.role != "client":
+            # Create a dummy mon
+            hostname = node.hostname
+            dummy_mon = f"sudo touch /var/lib/ceph/mon/ceph-{hostname}"
+            _, err = node.exec_command(cmd=f"{dummy_mon}")
+            if err:
+                log.error("Failed to create dummy mon")
+                return 1
+
+            # Try cephadm shell command to make sure the above dummy mon doesn't interfere
+            # Checking cluster health
+            status = cephadm.cluster.check_health(
+                rhbuild=config.get("rhbuild"), client=node
+            )
+            if status:
+                log.error(
+                    "Error!! Cephadm shell is trying to access config from the dummy mon"
+                )
+                return 1
+            log.info(
+                "Expected. Cephadm shell doesn't tries to infer the config from the dummy mon"
```

Comment:
Same as above

Reply:
Here, an expected log message is printed. The above created dummy mon should not cause a problem while using cephadm shell. IMO, this is clear log message

Reply:
This message seems ok to me @vamahaja